relayer/gravity_batch_relayer.go

```diff
@@ -0,0 +1,69 @@
+package relayer
+
+import (
+	"context"
+	"github.com/palomachain/pigeon/internal/liblog"
+	"sync"
+
+	"github.com/palomachain/pigeon/chain"
+	"github.com/palomachain/pigeon/util/slice"
+	log "github.com/sirupsen/logrus"
+)
+
+func (r *Relayer) GravityRelayBatches(ctx context.Context, locker sync.Locker) error {
+	log.Info("relayer loop")
```

Comment:
The lib can also be used here: 

```suggestion
	log := liblog.WithContext(ctx).Info("relayer loop")
```